### ISSUE TITLE
Call the necessary smart contract methods in the Hedera StreamProvider

### ISSUE THREAD

**Issue by Ali-Amin** [KarimElghamry/alvarium-sdk-go#3]
Assuming this smart contract is already created on the Hedera network

```solidity
/*
    this contract provides the ability for a trust provider to
    charge their users on bulk basis.
*/
contract AlvariumTaaS {
 ...
    // a mapping to store unbilled annotations with respect
    // to each user. This can/will be abstracted away in its own
    // view model for more efficiency.
    mapping(address => string[]) unbilledTransactions;

    // a method to record consensus service TOPIC_MESSAGE_SUBMIT transactions as unbilled 
    function publish(string memory transactionId) public {
        // record annotation as unbilled for the sending user
        unbilledTransactions[msg.sender].push(transactionId);
    }
...
}
```

The Hedera stream provider should call this method after the annotation is created.

**Comment by Ali-Amin** [comment]
Closed by #4.

**Comment by Ali-Amin** [comment]
Closed by #4.
